app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -45,10 +45,11 @@ def configuration_scripts
 
   def configuration_script_sources
     collector.projects.each do |project|
+
       inventory_object = persister.configuration_script_sources.find_or_build(project.id.to_s)
       inventory_object.description = project.description
       inventory_object.name = project.name
-      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s)
+      inventory_object.authentication = persister.credentials.lazy_find(project.credential_id.to_s) if project.credential
```

Comment:
lazy_find('') should resolve to nil, so it ends up doing inventory_object.authentication = nil, so if that won't crash, it should work :-)